index.ts

```diff
@@ -108,16 +102,15 @@ function getVideos(doc: any) {
   let videoObj;
   let index;
 
-  const nodes = doc(`meta[property='og:video']`);
-  const { length } = nodes;
+  const nodes = metaTag(doc, `og:video`, `property`) || metaTag(doc, `og:video`, `name`);
 
-  if (length) {
-    nodeTypes = doc(`meta[property='og:video:type']`);
-    nodeSecureUrls = doc(`meta[property='og:video:secure_url']`);
-    width = doc(`meta[property='og:video:width']`).attr(`content`);
-    height = doc(`meta[property='og:video:height']`).attr(`content`);
+  if (nodes) {
```

Comment:
also apply: `nodes?.length` here and we should be good to go

Reply:
done!